config/index.js

```diff
@@ -44,13 +44,16 @@ module.exports = {
     SECRET: process.env.API_SECRET,
     TIMEOUT: 30000, // in milliseconds
     CACHE_EXPIRY: 60 * 60 * 24 * 7, // in seconds (7 days)
-    MAX_FILE_UPLOAD_SIZE: 50 * 1024 * 1024, // 50 mega bytes
   },
   DATE_FORMATS: {
     SHORT: 'd M yyyy',
     LONG: 'd MMM yyyy',
     WITH_DAY: 'EEEE d MMM yyyy',
   },
+  FILE_UPLOADS: {
+    UPLOAD_DIR: '.tmp/uploads/',
+    MAX_FILE_SIZE: 50 * 1024 * 1024, // 50mb
```

Comment:
Could it be cautious to have a total max size? E.g. across all the uploaded documents.

Reply:
It doesn't seem like that can be set from the [multer limits config](https://github.com/expressjs/multer#limits).

We could set a maximum number of files that can be uploaded?

@jason-sims what are your thoughts on limiting the amount of files a user can upload during this process?

Reply:
@spikeheap might also be one for you ☝️